worlds/stardew_valley/regions.py

```diff
@@ -353,8 +354,81 @@ def inverted(self):
 ]
 
 
+# Attempt to implement remote loading of modded data if relevant (to avoid circular references and clutter)
+def loaded_mod_region_data(mod: str, data_type: str, world_options: StardewOptions):
+    loaded_mod_data = []
+    if mod in world_options[options.Mods]:
+        for mod_data in ModDataList:
```

Comment:
ModDataList could instead be a dictionary, where the key is the mod name, and the value is the mod data. This would allow you to simple get the mod you want from it, instead of looping checking the names one by one